api/app/Models/Classification.php

```diff
@@ -39,4 +40,15 @@ public function genericJobTitles(): HasMany
     {
         return $this->hasMany(GenericJobTitle::class);
     }
+
+    /**
+     * Get the classification display name, e.g. IT-01
+     */
+    protected function displayName(): Attribute
```

Comment:
Has this been used somewhere?

Reply:
Yeah, it's used to build the notification.  Gray said he wanted the longer form of the pool name in the notification.
https://github.com/GCTC-NTGC/gc-digital-talent/blob/1c1419cf935dd13158248b3f90c81acd9e99a225/api/app/Console/Commands/SendNotificationsApplicationDeadlineApproaching.php#L68